frontend/src/app/main/ui/workspace/sidebar/sitemap.cljs

```diff
@@ -82,10 +82,9 @@
 
         on-drop
         (mf/use-callback
-         (mf/deps id)
-         (fn [side {:keys [id] :as data}]
-           (let [index (if (= :bot side) (inc index) index)]
-             (st/emit! (dw/relocate-page id index)))))
+         (mf/deps id index)
+         (fn [_side {:keys [id] :as data}]
+           (st/emit! (dw/relocate-page id index))))
```

Comment:
Don't understand this change. The check about side is for user to choose if they want to drop *after* or *before* the page they are pointing at. Why eliminate it?